### PR TITLE
Update New-CsTeamsUnassignedNumberTreatment.md

### PR DESCRIPTION
**Pull request by JensTrier**
Adding comment on assigned numbers in unassigned number range on New cmdlet and changing -Filter description on Get cmdlet

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsTeamsUnassignedNumberTreatment.md [hunk 1]
```diff
@@ -40,12 +40,17 @@ Get-CsTeamsUnassignedNumberTreatment
 ```
 This example displays all configured treatments.
 
+### Example 3
+```powershell
+Get-CsTeamsUnassignedNumberTreatment -Filter Ann*
+```
+This example displays all configured treatments with an Identity starting with Ann
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsUnassignedNumberTreatment.md:47]:
```suggestion
This example displays all configured treatments with an Identity starting with Ann.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**